code/modules/mob/living/emote.dm

```diff
@@ -431,19 +431,19 @@
 	emote_type = EMOTE_VISIBLE | EMOTE_AUDIBLE
 	vary = TRUE
 
-/datum/emote/living/sniff/get_sound(mob/living/carbon/human/user)
-	if(!istype(user))
-		return
-	return user.dna.species.get_sniff_sound(user)
-
-
 /datum/emote/living/snore
 	key = "snore"
 	key_third_person = "snores"
 	message = "snores."
 	message_mime = "sleeps soundly."
 	emote_type = EMOTE_VISIBLE | EMOTE_AUDIBLE
 	stat_allowed = UNCONSCIOUS
+	mob_type_blacklist_typecache = list(/mob/living/carbon/human)
+
+/datum/emote/living/sniff/get_sound(mob/living/carbon/human/user)
+	if(!istype(user))
+		return
+	return user.dna.species.get_sniff_sound(user)
```

Comment:
while moving snore back, accidentally moved `/datum/emote/living/sniff/get_sound` below `/datum/emote/living/snore` instead of `/datum/emote/living/sniff`